ipalib/__init__.py

```diff
@@ -942,12 +942,17 @@ def packages(self):
                 ipaserver.plugins,
             )
         else:
-            import ipaclient.remote_plugins
-            import ipaclient.plugins
-            result = (
-                ipaclient.remote_plugins.get_package(self),
-                ipaclient.plugins,
-            )
+            try:
+                import ipaclient.remote_plugins
+                import ipaclient.plugins
+                result = (
+                    ipaclient.remote_plugins.get_package(self),
+                    ipaclient.plugins,
+                )
+            except NetworkError:
+                #  instead of raising the default error connection message,
+                #  raising a more user-friendly one
+                raise GenericError('Cannot find IPA server to contact')
```

Comment:
I wasn't trying to just change the message, but check if IPA is well configured. I thought the message `can't connect to http://localhost:8888/ipa/json` was saying that.

I have checked the PR#25; to fix independent scripts seems easier than apply the same  behavior to all commands. Since the changes could be done locally. 

I was thinking in to do something like in [ipalib/plugable.py](https://github.com/freeipa/freeipa/blob/master/ipalib/plugable.py#L706-L715):
```python
    from ipaserver.install import installutils

    def finalize(self):
        """
        Finalize the registration, instantiate the plugins.

        `API.bootstrap` will automatically be called if it hasn't been
        already.
        """
        
        installutils.check_server_configuration()

        self.__doing('finalize')
        self.__do_if_not_done('load_plugins')
```

However, it doesn't seems a good idea import ipaserver code in ipalib.